core/src/test/resources/org/bitcoinj/script/script_tests.json

```diff
@@ -1119,5 +1121,1489 @@
 
 ["0x00", "'00' EQUAL", "P2SH,STRICTENC", "EVAL_FALSE", "Basic OP_0 execution"],
 
+["MINIMALDATA enforcement for PUSHDATAs"],
```

Comment:
If this block, please only copy the CSV-related tests.

(e.g. MINIMALDATA would go into a different PR that fixes #1396, but don't worry about that in the context of this PR.)